src/react-components/src/components/03-organisms/Hero/Hero.tsx

```diff
@@ -0,0 +1,119 @@
+// OH-30 Header with Image Right
+import * as React from "react";
+import bem from "../../../utils/bem";
+import Image from "../../01-atoms/Images/Image/Image";
+import Heading from "../../01-atoms/Text/Headings/Heading";
+import { HeroTypes } from "./HeroTypes";
+
+export interface HeroProps {
+  /** Can be Primary, secondary, tertiary, or 50/50.
+    */
+  heroType?: HeroTypes;
+
+  /** To be deleted after Crystal's ID merge
```

Comment:
ID generation is gonna get punted, the Filters ticket is large enough as is.

Reply:
Updated the variable description!